WeatherApp/Models/WeatherList/CoreDataModels/CurrentForecastEntity+CoreDataClass.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  CityForecastEntity+CoreDataClass.swift
+//  WeatherApp
+//
+//  Created by Lucija Balja on 22/08/2020.
+//  Copyright © 2020 Lucija Balja. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+
+public class CurrentForecastEntity: NSManagedObject {
+
+    class func createFrom(_ currentWeatherResponse: CurrentWeatherResponse, context: NSManagedObjectContext) {
+        guard let _ = loadCurrentForecast(context: context) else {
```

Comment:
if you are not assigning value, use `guard loadCurrentForecast(context: context) != nil`